ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/TransactionReceipt.java

```diff
@@ -103,6 +124,14 @@ private TransactionReceipt(
    * @param out The RLP output to write to
    */
   public void writeTo(final RLPOutput out) {
+    writeTo(out, false);
+  }
+
+  public void writeToWithReason(final RLPOutput out) {
```

Comment:
does this need to exist? - or do we just expose the following function?
If we keep this, maybe rename to writeToWithRevertReason (i.e. include the word revert).

Reply:
renamed